openedx/edx-ora2
# fix ESG link behavior from instructor dashboard when ESG flag enable

**TL;DR -** Fix bug for openassessment title is leading to ESG link instead of ORA.

JIRA: [AU-576](https://openedx.atlassian.net/browse/AU-576)

**What changed?**

- Revert the change of the older URL logic
- Create ESGCell logic and stop reusing older logic that is overlapping with other column as well.
- Update unit test
- **Minor** `load_static` to help with hot reload. I didn't intend it to be here but accidentally commit it already.

**Developer Checklist**


**Testing Instructions**

make sure [ENABLE_XBLOCK_VIEW_ENDPOINT](https://github.com/openedx/edx-platform/blob/29e50710dc2b59ad73fa5234e32b07242a6f0411/lms/envs/common.py#L227) is set to true.

- Disabled ESG
  - Go to [http://localhost:18000/admin/waffle/flag/](http://localhost:18000/admin/waffle/flag/)
  - add/update Flag
    - name `openresponseassessment.enhanced_staff_grader`
    - change everyone to `no`
    - check `staff`
    - Save
  - [ ] Go to `instructor dashboard -> Open Responses`
  - [ ] There shouldn't be additional column
  - [ ] Staff column link should lead to in-window legacy staff grader
  - [ ] Assessment column link should lead to in-window ora 
- Enabled ESG
  - Go to [http://localhost:18000/admin/waffle/flag/](http://localhost:18000/admin/waffle/flag/)
  - add/update Flag
    - name `openresponseassessment.enhanced_staff_grader`
    - change everyone to `yes`
    - check `staff`
    - Save
  - [ ] Go to `instructor dashboard -> Open Responses`
  - [ ] There should be additional column title response with link to new page of ESG
  - [ ] Staff column link should lead to new page of ESG
  - [ ] **Previous bug**: Assessment column link should remain link to in-window ora

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:


FYI: @openedx/content-aurora
